src/schema/search/SearchEntity.ts

```diff
@@ -30,6 +30,9 @@ export const SearchEntity = new GraphQLEnumType({
     GENE: {
       value: "Gene",
     },
+    INSTITUTION: {
+      value: "institution",
```

Comment:
Are you using case to differentiate between "real" types and types derived from others? For example, could we have used "Institution" and "Gallery" values instead of "institution" and "gallery"?

Reply:
We can't use those values, b/c the corresponding Gravity PR accepts `?indexes[]=gallery` (lowercase).

Yea- I guess I was using lower case since these are 'synthetic' types, but no real convention!

Reply:
Gotcha. Right, I was thinking of the Gravity implementation as well. I lean towards isolating that distinction as an implementation detail within Gravity and in Metaphysics, treat it like any other type with the same casing. Minor curious thought though. Not blocking :+1: